### DISCUSSION TITLE
clr-alert only works with core icon sets?

### DISCUSSION THREAD

**Discussion by Shakiem** [vmware-clarity/ng-clarity#193] [Q&A] [answered]
I just upgraded clr/angular to 13.3.1 from 1.3.9 and I am noticing that the <clr-alert/> component can only load icons from the core icon set with the clrAlertIcon attribute.

Is there some other setup required for this to work with all icons? I had no issues in 1.3.9 using icons from the technology set.

Below shows a test setup where I am displaying two app level alerts, one with an icon from the technology shapes and another with an icon from the core shapes. Only the core shape displays correctly.

![clr-alert-13](https://user-images.githubusercontent.com/7884554/175651232-91c49682-546d-41b8-8302-0da78d008154.png)

**Comment by kevinbuhmann** [comment, marked as answer]
https://github.com/vmware-clarity/ng-clarity/issues/204#issuecomment-1171406594
